stories/SQFormDropdown.stories.tsx

```diff
@@ -70,7 +70,11 @@ const Template: DropdownStoryType = (args) => {
         validationSchema={schema}
         {...sqFormProps}
       >
-        <SQFormDropdownComponent {...dropdownProps} size={getSizeProp(size)} />
+        <SQFormDropdownComponent
```

Comment:
@malikfarhan183 Looks really good.  Instead of replacing the current story, I think we should add a new story with the helper text.  That way, we would have an example of both a dropdown with the informational text, and an example without it.

Reply:
@aharpt Thanks for your great suggestion. I have created a new story for this new helper text prop. 
![image](https://user-images.githubusercontent.com/42131897/235916961-5d357846-c801-49dc-b87c-027e673ac2ba.png)